exchanges.yml

```
@@ -386,20 +386,39 @@ components:
       additionalProperties: false
       description: Object containing information about an active exchange.
       properties:
-        exchangeId:
-          type: string
-          description: The URL that uniquely identifies the exchange.
-        ttl:
-          type: string
-          description: The time to live for the exchange.
-        nextStep:
-          type: string
-          description: The interaction the exchange is expecting to happen next.
-        completedSteps:
-          type: array
-          description: The steps completed by the exchange previously.
-          items:
-            type: string
+        exchange:
+          type: object
+          description: The active exchange.
+          properties:
+            exchangeId:
+              type: string
+              description: The URL that uniquely identifies the exchange.
+            ttl:
+              type: string
+              description: The time to live for the exchange.
+            nextStep:
+              type: string
+              description: The interaction the exchange is expecting to happen next.
+            completedSteps:
+              type: array
+              description: The steps completed by the exchange previously.
+              items:
+                type: string
+            state:
+              type: string
+              description: The status ("pending" | "complete" | "invalid") of the exchange, set to "pending" on creation.
+            variables:
+              type: array
+              description: Template variables to be used in the exchange.
+              items:
+                type: object
+                properties:
+                  type:
+                    type: string
+                    description: The type of template.
+                  template:
+                    type: string
+                    description: The template itself.
```

This is actually a map. Probably needs fixing in the exchange.yml as well.

```suggestion
            variables:
              type: object
              description: Template variables to be used in the exchange.
```

---

I think you mean in `CreateExchangeResponse`? There is no `exchange.yml` that I could find.